BundleClient/app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,7 @@
         android:label="@string/app_name"
         android:theme="@style/Base.V7.Theme.AppCompat.Light" >
         <activity
-            android:name=".HelloworldActivity"
+            android:name="net.discdd.bundleclient.HelloworldActivity"
```

Comment:
if you change the package on line 3 to net.discdd.bundleclient I don't think you need this change.

Reply:
i'm wrong, we are not supposed to use the package anymore.

Reply:
Removed it.